src/commands/build.ts

```diff
@@ -255,6 +255,8 @@ export async function command(commandOptions: CommandOptions) {
           isDev: false,
         });
         if (!builtFile) {
+          let {id} = workerConfig;
+          messageBus.emit('WORKER_COMPLETE', {id, error: 'Build Failed!'});
```

Comment:
I don't think that this is actually considered an error today. I'm worried that this would break existing apps.